src/libraries/System.Drawing.Common/src/System/Drawing/Drawing2D/GraphicsPathIterator.cs

```diff
@@ -13,7 +13,7 @@ public sealed class GraphicsPathIterator : MarshalByRefObject, IDisposable
         public GraphicsPathIterator(GraphicsPath? path)
         {
             IntPtr nativeIter = IntPtr.Zero;
```

Comment:
Should this be ``SafeGraphicsPathIteratorHandle``?

Reply:
Didn't port it yet. The PR is still in progress and not yet ready for review.